config/locales/de.yml

```diff
@@ -154,29 +201,72 @@ de:
         mailer:
           invitation_instructions:
             accept_until_format: "%d. %B %Y %H:%M"
+  gender:
+    female: Weiblich
+    male: Männlich
+  language_level:
+    native_speaker: Muttersprachler
+    fluent: Fließend
+    good: Gut
+    basic: Elementar
+  role:
+    superadmin: Superadmin
+    admin: Admin
+    social_worker: Sozialarbeiter
```

Comment:
In future kids they use that form: Sozialarbeiter/in. 
I don't know which is more correct, so I'll trust you and @nuritnt on which one to use.